autogen/cluster_regional.tf

```diff
@@ -103,6 +95,7 @@ resource "google_container_cluster" "primary" {
 {% endif %}
 
   remove_default_node_pool = "${var.remove_default_node_pool}"
```

Comment:
I'm wondering if we should also start setting remove_default_node_pool to `true` by default?

Reply:
I tend to think we should default it to `true`. Especially with how minimal the possible configuration is for the default node pool, I don't know that it makes sense to promote using it.

Reply:
Agreed, let's consider changing that in a future PR.